packages/transports/http/src/index.ts

```diff
@@ -14,7 +14,9 @@ export type HTTPTransportOptions<
   HTTPExecutorOptions,
   'useGETForQueries' | 'method' | 'timeout' | 'credentials' | 'retry'
 > & {
-  subscriptions?: TransportEntry<TSubscriptionTransportOptions>;
+  subscriptions?: TransportEntry<
+    TSubscriptionTransportOptions | ((execReq: ExecutionRequest) => TSubscriptionTransportOptions)
+  >;
```

Comment:
This should be in changesets.

Reply:
Not sure it will stay if we move to make `prpagateHaders` just work on ws